### ISSUE TITLE
Move technical discussion to other forum

### ISSUE THREAD

**Issue by ilblackdragon** [near/phase2-validator-votes#7]
As NEAR starts evolving into community governed network starting with this more technical questions around Phase 2, I think it's important to figure out an accessible medium.

I find discussions in github in PRs somewhat limiting (and especially they are limiting for non developers, who are usually don't have a github account). And discussions in Telegram ends up been non permanent and hard to make decisions.

There are two options to move:
 - Discourse (we had one at nearresear.ch before)
 - https://commonwealth.im/near/proposal/discussion/646-use-commonwealth-for-technical-governance-discussions

Commonwealth has benefit of the ability to integrate to the chain itself, but it seemed like Discourse has plugins for various stuff as well.

Feel free to react with 👍  for Discourse, 💯 or 🎉    for Commonwealth, 🔢 or 🚀  to stay on Github.

**Comment by gaia** [comment]
hmmm these are the only available reactions to me 
![image](https://user-images.githubusercontent.com/87547/95244950-53c15e80-07c7-11eb-8c08-75a25bfada96.png)

**Comment by evgenykuzyakov** [comment]
🔢

**Comment by kucharskim** [comment]
I'm like @gaia, only few icons to chose from no `1234` icon nor `100`. I know only github and email, would need to look into those other tools.

**Comment by ilblackdragon** [comment]
@gaia @kucharskim updated with alternatives. Github is a bit weird what it shows I guess when typing.

**Comment by mattlockyer** [comment]
:+1: Discourse is well known for being great for long form discussion.
:rocket: GH issues are also ok, but I feel somewhat less inclusive of "non-developers".

**Comment by gaia** [comment]
so we are supposed to make a new comment to include our reply? ilya, do you have those icons in the drop down menu?

**Comment by mattlockyer** [comment]
> I'm confused. and I like GH better than Discord.

I believe we're debating Discourse (forums) not Discord.

**Comment by NAKsir-melody** [comment]
Discourse LGTM.

**Comment by Mike-LunaNova** [comment]
Thank you for initiating this useful discussion. 

To stay focused on your precise options:

- Discourse 
- Commonwealth

The Github feature of being able to quote seems powerful whereas, as far as I can tell Commonwealth seems to be able to index the discussions which is useful. The ability for Commonwealth to integrate to the chain itself,could be very powerful for debates and voting at some point. That is not to say that Discourse plugins may not be able to fulfill this requirement.

I think the most important aspect is the participation we as a community have and the discipline we adopt in using whichever forum. It may be appropriate to have forum moderator(s). However I see that Discourse seem to offer an automatic trust system which may obviate this.

A number have, understandably, confused discourse with discord.  This is an option for aspects of community communication, but from experience seems to be very chat-focused meaning that any debate can be interrupted and indeed at times hard to find.

In conclusion both seem suitable to me. However the difficulty even in this thread for people to be able to reliably convey their preference must be addressed.

A final thought - the question must relate to ANY conversation being held by the community about aspects of NEAR and as such is not just limited to technical aspects.

**Comment by auxelarry** [comment]
Discourse

**Comment by insideNIMA** [comment]
Discord and Discourse can co-exists. It's best to move long form valuable conversations to Discourse/forum. 

Both are also compatible with sourcecred.io 🚀

**Comment by crasskitty** [comment]
I would like to point out that some community members are already on portal.near.org which is running on Tribe:

- Tribe is completely searchable.
- Tribe also has the potential of us being able to pay people on it directly with NEAR for answering questions and contributing to the community.
- Tribe also has built in polling/voting
- Tribe could work well for both NEAR core and NEAR community as it is one portal that can easily be divided into many types of groups that are easy to keep track of.
- Tribe is currently closed but once we open it to everyone, which yes, is an option, will be completely SEO searchable.
- We can certainly ask for the question feature.
- Tribe has built in amazing analytics
- We already transferred the community too many times, why are we trying to solve it if it isn't broken? If we go with the actual plan we had for tribe, we start rolling people out to it in stages until everyone is on it. Which then accomplishes the goal.


 Basically, everything you want we can do with Tribe & Tribe is already something 200+ community members have been trying.
